src/games/strategy/engine/framework/startup/ui/MetaSetupPanel.java

```diff
@@ -287,6 +276,7 @@ private void connectToLobby() {
       return Optional.empty();
     }
     Yaml yaml = new Yaml();
+    @SuppressWarnings("unchecked")
```

Comment:
Same, please remove the `@SuppressWarnings` unless the object is actually used (best solution is ot remove the unused object, failing that, leave the warning so we come back to it one day, hiding a problem is not a good thing)